doc/docs/modules/ROOT/pages/quickstart.adoc

```diff
@@ -8,28 +8,274 @@ Get started fast for common scenarios, using Neo4j Streams plugin or Kafka Conne
+|No
+
+|`connection.liveness.timeout.msec`
+|Liveness check timeout
+|(Neo4j Driver default)
+|No
+
+|`connection.acquisition.timeout.msecs`
+|Connection acquisition timeout in milliseconds
+|(Neo4j Driver default)
+|No
+
+|`connection.timeout.msecs`
+|Connection timeout in milliseconds
+|(Neo4j Driver default)
+|No
+
+
+4+|*Session Options*
+
+|`database`
+|Database name to connect to
+|``
+|No
+
+|`access.mode`
+|Access mode
+|`write`
+|No
+
+4+|*Data Read Options*
+
+|`query`
+|Specify a query to read the data
+|``
+|Yes^*^
+
+|`labels`
+|List of labels separated by `:`. First label is the primary label
+|``
+|Yes^*^
+
+|`relationship`
+|List of labels separated by `:` (first label is the primary label)
+|``
+|Yes^*^
+
+|`schema.flatten.limit`
+|Number of records to be used to create the Schema (only if APOC are not installed)
+|`10`
+|No
+
+|===
+
+^*^ Just one of the options can be specified.
+
+== Read Data
+
+Reading data from a Neo4j Database can be done in 3 ways:
+
+ * with a Cypher query
+ * with a set of node Labels 
+ * by specifying a relationship
+
+=== Considerations on the schema
+
+Spark works with data in a tabular fixed schema. To accomplish this Neo4j Connector has a schema infer system that creates the schema based on the data requested for the read. Each read data method has is own strategy to create it, that will be explained it each section.
+
+TK list of supported data types
+
+==== Complex Data Types
+
+Spark doesn't support all Neo4j data types (ie: Point, Time, Duration). Such types are transformed into Struct types containing all the useful data.
+
+|===
+
+|Type |Struct 
+
+|`Duration`
+a|
+----
+Struct(Array(
+    ("type", DataTypes.StringType, false),
+    ("months", DataTypes.IntegerType, false),
+    ("days", DataTypes.IntegerType, false),
+    ("seconds", DataTypes.IntegerType, false),
+    ("nanoseconds", DataTypes.IntegerType, false),
+    ("value", DataTypes.StringType, false)
+  ))
+----
+
+|`Point`
+a|
+----
+Struct(Array(
+    ("type", DataTypes.StringType, false),
+    ("srid", DataTypes.IntegerType, false),
+    ("x", DataTypes.DoubleType, false),
+    ("y", DataTypes.DoubleType, false),
+    ("z", DataTypes.DoubleType, true),
+  ))
+----
+
+|`Time`
+a|
+----
+Struct(Array(
+    ("type", DataTypes.StringType, false),
+    ("value", DataTypes.StringType, false)
+  ))
+----
+
+|=== 
+
+=== Read data by Node Labels
+
+You can both specify a single label, like this example
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+        .option("url", "bolt://localhost:7687")
+        .option("labels", "Person")
+        .load()
+
+df.show()
+```
+
+Multiple labels can be specified, separated by `:`
+```scala
+val df = spark.read.format("org.neo4j.spark.DataSource")
+        .option("url", "bolt://localhost:7687")
+        .option("labels", "Person:Customer:Admin")
+        .load()
+
+df.show()
+```
+
+[NOTE]
+Remember that the first Label is considered to be the primary label.
```

Comment:
I don't know what "Primary label" means, or what impact it has.

Reply:
It' just how Cypher DSL handles the labels, we can remove this.